project-base/app/config/parameters_google_cloud.yml.dist

```diff
@@ -0,0 +1,3 @@
+parameters:
+    google_cloud_project_id: ~
+    google_cloud_storage_bucket_name: ~
```

Comment:
Config file templates (`parameters.yml.dist` or `parameters_<ENV>.yml.dist`) are mentioned in `composer.json` option `extra.incenteev-parameters`, so users are prompted to fill them out during `composer install`. Is it because it's an unusual use-case?